datadog_checks_base/datadog_checks/base/utils/db/sql.py

```diff
@@ -13,17 +15,19 @@
 
 
 def compute_sql_signature(normalized_query):
+    # type: (str) -> Optional[str]
     """
     Given an already obfuscated & normalized SQL query, generate its 64-bit hex signature.
     """
     if not normalized_query:
         return None
     # Note: please be cautious when changing this function as some features rely on this
     # hash matching the APM resource hash generated on our backend.
-    return format(mmh3.hash64(normalized_query, signed=False)[0], 'x')
+    return format(mmh3.hash64(normalized_query, signed=False)[0], str('x'))
```

Comment:
> `str('x')`

This looks weird, what is the motivation for this syntax?

Reply:
Because I'm using `from __future__ import unicode_literals`, the 'x' is a unicode string (py2) being passed to format, which only accepts ascii `str` as an argument.

Reply:
Okay! So just to be sure, this is unrelated to typing, correct?

Reply:
Oh, no it is due to typing. The error is from mypy, but does not produce a code execption in py2.